assets/javascripts/discourse/widgets/qa-post.js.es6

```diff
@@ -23,7 +23,7 @@ export default createWidget('qa-post', {
     const user = this.currentUser;
 
     if (post.get('topic.voted')) {
```

Comment:
We'll need to change this conditional to check our new ``can_vote`` property instead of the old ``voted`` property.

Reply:
Given that the previous `topic.voted` didn't seem to pass any user argument in the javascript, but the serializer seems to get it from the scope:

```ruby
  def voted
    scope.current_user && ::Topic.voted(object.topic, scope.current_user)
  end
```
I'm also going to assume that I can do similar with `topic.can_vote`:

```ruby
    if (post.get('topic.can_vote') == false) {
      return bootbox.alert(I18n.t('vote.user_over_limit'));
    }
```
But you'd probably want me to use the same convention, `!`
```javascript
    if (!post.get('topic.can_vote')) {
      return bootbox.alert(I18n.t('vote.user_over_limit'));
    }
```